ui/src/router/components/form/components/router_config/RulesPanel.js

```diff
@@ -34,7 +28,7 @@ export const RulesPanel = ({
   };
 
   const addRuleButton = (
-    <EuiButton fullWidth onClick={onAddRule} isDisabled={routes.length < 2}>
```

Comment:
Same change as above - previously, we always had a default route and no traffic rule could be created on it.